pihole/datadog_checks/pihole/pihole.py

```diff
@@ -0,0 +1,97 @@
+import requests
+
+from datadog_checks.base import AgentCheck, ConfigurationError
+
+
+class PiholeCheck(AgentCheck):
+    @staticmethod
+    def _collect_response(url):
+        response = requests.get(url)
```

Comment:
Do not use `requests` directly. Use `self.http.get`instead. When you do you can inclue `template: instances/http` on the config specs